src/Task.ts

```diff
@@ -132,6 +134,7 @@ export class Task {
     public readonly scheduledDate: Moment | null;
     public readonly dueDate: Moment | null;
     public readonly doneDate: Moment | null;
+    public readonly createdDate: Moment | null;
```

Comment:
There are enough steps needed when adding a new field to Tasks that I wrote up a page on the Contributing guide for it:
<https://publish.obsidian.md/tasks-contributing/Code/How+do+I+add+a+new+field+to+the+Task+class>
Please follow its steps for updating `Task.identicalTo()`

Reply:
Add test, change method identicalTo.